{"name": "test_b", "status": "failed", "statusDetails": {"message": "AssertionError: assert False", "trace": "self = <cases.test_class.TestClass object at 0x000001B22E6E54E0>\n\n    @pytest.mark.dependency()\n    def test_b(self):\n        print(\"方法2\")\n>       assert False\nE       assert False\n\ntest_class.py:14: AssertionError"}, "start": 1617002194488, "stop": 1617002194488, "uuid": "6dec9e51-0d3d-48af-8a15-2ae7fd2b2453", "historyId": "3666464962ef9b3194d0de8046ac980f", "testCaseId": "9ec1ce2de8879a24acf44e8277658695", "fullName": "cases.test_class.TestClass#test_b", "labels": [{"name": "tag", "value": "dependency"}, {"name": "parentSuite", "value": "cases"}, {"name": "suite", "value": "test_class"}, {"name": "subSuite", "value": "TestClass"}, {"name": "host", "value": "DESKTOP-MI5M7JM"}, {"name": "thread", "value": "8800-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "cases.test_class"}]}